components/raftstore/src/store/peer.rs

```diff
@@ -2006,6 +2011,15 @@ impl Peer {
                 box_err!("{} can not read index due to no leader", self.tag),
             );
             poll_ctx.raft_metrics.invalid_proposal.read_index_no_leader += 1;
+            // The leader may be hibernated, send a message for trying to awaken the leader.
+            if poll_ctx.cfg.hibernate_regions
+                && (self.bcast_wake_up_time.is_none()
+                    || self.bcast_wake_up_time.as_ref().unwrap().elapsed()
+                        >= Duration::from_millis(MIN_BCAST_WAKE_UP_INTERVAL))
+            {
+                self.bcast_wake_up_message(&mut poll_ctx.trans);
+                self.bcast_wake_up_time = Some(Instant::now());
```

Comment:
How about using `now_coarse`?

Reply:
For now `raftstore` only uses `std::time::Instant` rather `tikv_util::time::Instant`, we could repalce it later.

Reply:
Why not solve it now? I worry about the performance impact. Note that the timer is called not just for learner but also followers.

Reply:
Is there any risk to replace all of the `std::time::Instant` into `tikv_util::time::Instant`?